### PR TITLE
Feature/pxblue 2546 angular global action list

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 2546 .

#### Changes proposed in this Pull Request:
- Global Action list route under List.
- Desktop view
- Mobile View

#### Screenshots:

Screenshot 2021-11-02 at 11 57 02 AM
 

Screenshot 2021-11-02 at 11 57 22 AM

#### To Test:
- http://localhost:4200/lists/in-panel-header

### CODE REVIEW

**Comment by ektasawant** [comment]
Created issue [44](https://github.com/pxblue/angular-themes/issues/44) for matching background of mat-select active item.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -142,6 +143,11 @@ export const ROUTES: DrawerItem[] = [
                 path: 'action-list',
                 component: ActionListComponent,
             },
+            {
```
Comment (ektasawant) [inline src/app/app-routing.module.ts:146]: Added new route for global action list with panel header. (Not matching with react). 

Angular design
Screenshot 2021-11-02 at 12 22 27 PM

React Design:
Screenshot 2021-11-02 at 12 23 23 PM

**Comment by emclaug2** [comment]
Taking a look at this now;  is the padding at the bottom of this menu expected? It seems weird to me since there's no top-padding: 
image

**Comment by emclaug2** [comment]
Overall this works great; I'm happy with the code.  Just see comment above.

**Review by emclaug2 — APPROVED** [review]

**Comment by ektasawant** [comment]
> Overall this works great; I'm happy with the code. Just see comment above.

Fixed height issue. Please review.
Screenshot 2021-11-02 at 8 24 13 PM

**Comment by emclaug2** [comment]
I'll leave this open for a couple hours to give others time to review; if no comments I'll merge it in.  Thanks @ektasawant

**Comment by ektasawant** [comment]
> I'll leave this open for a couple hours to give others time to review; if no comments I'll merge it in. Thanks @ektasawant

Sure 👍 .

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (jeffvg) [reply src/app/app-routing.module.ts:146]: we called this out in react. There are plans to rename parent dir for both

**Review by daileytj — APPROVED** [review]

**Review by jeffvg — APPROVED** [review]

**Pull request merged**